lit/src/services/firebase_service.ts

```diff
@@ -23,7 +23,7 @@ import {
     FIREBASE_LOCAL_HOST_PORT_FIRESTORE,
     FIREBASE_LOCAL_HOST_PORT_FUNCTIONS
 } from '../shared/constants';
-import { FIREBASE_CONFIG } from '../shared/config';
+import { FIREBASE_CONFIG } from '../shared/config_example';
```

Comment:
You should not import the example config, but copy it to the example file (as indicated in the lit readme). Léo and I are working with the project deployed at EPFL and we need the code to be importing our own config.

Reply:
+1

Reply:
oh yes, whoops! an accidental add, let me revert this one. thank you for catching